go-md2man.1.md

```diff
@@ -13,6 +13,18 @@ written purely in Go so as to reduce dependencies on 3rd party libs.
 
 By default, the input is stdin and the output is stdout.
 
+# OPTIONS
```

Comment:
I realize I should probably include the generated roff in the repo as well so we can see the changes.